release-notes/major9/09.19.0/valtimo-backend-libraries.md

```diff
@@ -19,9 +19,10 @@ The following features were added:
 
 The following bugs were fixed:
 
-* **Bug1**
+* **JdbcProcessFormAssociationRepository insert statement runtime exception**
 
-  Description of what the issue was.
+  Running on an externally hosted Mysql 8.0.15 instance caused unknown column type exceptions with useServerPrepStmts [see](https://github.com/brettwooldridge/HikariCP/wiki/MySQL-Configuration).
```

Comment:
Can you surround `useServerPrepStmts` with backticks?
Also, maybe rephrase the [see] part to:
`For more informtion, see [here](https://github.com/brettwooldridge/HikariCP/wiki/MySQL-Configuration).`